lib/common/netcdf-check.sh

```diff
@@ -165,8 +165,8 @@ trigger_checkers_and_add_signature() {
     trigger_checkers $file $backup_recipient $@
 
     if [ ${#@} == 0 ]; then
-        # no compliance checks triggered, so no signature
-        echo $file
+        # no compliance checks triggered
```

Comment:
Would be good to update the function description above to say that it will always make a writable temp copy of the file and return its full path, and add checker signature if any checks were requested and passed.